server/models/services/CollectionJobService.js

```diff
@@ -350,6 +355,12 @@ const scheduleCollectionJob = async (
         COLLECTION_JOB_ATTRIBUTES,
         options
     );
+
+    if (process.env.AUTOMATION_CALLBACK_FQDN) {
```

Comment:
To reduce the potential for discrepancies between development and production, I'd prefer to increase parity between the code paths. Here, that would mean a consistent sequence of job creation followed by collection scheduling.

Reply:
I agree. I'll make sure to reduce the switching around of path/order on mock test/real automation